dotcom-rendering/src/amp/components/Ad.tsx

```diff
@@ -116,18 +109,9 @@ const useRealTimeConfig = (
 		});
 	}
 
-	// Control / AB testing not enabled
-
-	const prebidURL = [
-		// The tag_id in the URL is used to look up the bulk of the request
-		// In this case it corresponds to the placement ID of the bid requests
-		// on the prebid server
-		`${preBidServerPrefix}?tag_id=${placementId}`,
-		...commonPrebidUrlParams,
-	].join('&');
+	// Not in test - dont't run Prebid
 
 	return realTimeConfig({
-		url: usePrebid ? prebidURL : undefined,
```

Comment:
If we turn off the test via a switch this will now turn off Prebid entirely. Whilst the test is still running we won't encounter this condition, as pages will be placed into the two variants above.

Reply:
> If we turn off the test via a switch this will now turn off Prebid entirely

This would be bad right? ..._right?_  :)

Reply:
Not as bad as on the web!

But yes, it'd have an impact. Still this feels like the best "safe state" option to go with in the event we needed to stop running the test immediately.